.github/workflows/validate.yml

```diff
@@ -57,7 +57,7 @@ jobs:
           fetch-depth: 2
       - uses: actions/setup-go@v6
         with:
-          go-version: "${{ env.GO_VERSION }}"
+          go-version: stable
```

Comment:
Could you add a comment line to explain why `GO_VERSION` is discarded only here

Reply:
done